.travis.yml

```diff
@@ -98,10 +91,6 @@ script:
   # Lint the PHP files against parse errors.
   - if [[ "$LINT" == "1" ]]; then if find . -path ./vendor -prune -o -name "*.php" -exec php -l {} \; | grep "^[Parse error|Fatal error]"; then exit 1; fi; fi
 
-  # Validate the composer.json file.
-  # @link https://getcomposer.org/doc/03-cli.md#validate
-  - if [[ "$LINT" == "1" ]]; then composer validate --no-check-all --strict; fi
-
   # Run the unit tests.
   - |
     if [[ ${TRAVIS_PHP_VERSION:0:3} > "7.1" ]]; then
```

Comment:
This could all really be in separate script files rather than inline in `.travis.yml`.